tslint-server/src/server.ts

```diff
@@ -361,8 +366,9 @@ function doValidate(conn: server.IConnection, document: server.TextDocument): se
 
 	let result: tslint.LintResult;
 	try { // protect against tslint crashes
-		let tslint = new linter(fsPath, contents, options);
-		result = tslint.lint();
+		let tslint = new linter(options);
```

Comment:
The `options` come from the user's settings in VS Code. The only option that is currently used is the `rulesDirectory`

```typescript
options.rulesDirectory = settings.tslint.rulesDirectory || null;
```

Makes sense?